shared/src/commonMain/kotlin/com/example/travelapp_kmp/listing/ListScreenViewModel.kt

```diff
@@ -73,6 +98,20 @@ class ListScreenViewModel {
             is ListScreenState.Success -> state
         }
     }
+
+
+    override fun sortByNameAsc() {
+        fetchCountries(sortOrder = SortOrder.ASCENDING)
+    }
+
+    override fun sortByNameDesc() {
+        fetchCountries(sortOrder = SortOrder.DESCENDING)
+    }
+}
+
+enum class SortOrder {
+    ASCENDING,
+    DESCENDING
 }
```

Comment:
No need for interface i believe. We will send action from View to View Model
```

sealed interface Action{
      object SortAcending: Action
      object SortDecending:Action
}
```

Something like that. We will get action from view and we will handle that in view model. wdyt ?